sql/2019/05_Third_Parties/05_07.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Top 100 third party domains by total byte weight
+SELECT
+  thirdPartyDomain,
```

Comment:
The top thirdPartyDomain is null, do you want to filter that out?

Reply:
I guess we already have the total byte weight 1p vs. 3p comparison, so yeah SG 👍